roles/jenkins_common/tasks/main.yml

```diff
@@ -1,28 +1,30 @@
 - name: install beaker client
+  become: true
```

Comment:
If the entire file is become: true, we might decide rather to place become: true at the level of site.yml for this. But we'd have to be OK with that propagating down to dependency roles included from meta/main.yml